new_ui/static/css/components.css

```
@@ -0,0 +1,480 @@
+}
+
+
+/* ================================================================
+ * SLIDERS (range input)
+ * ================================================================ */
+
+.slider {
+    -webkit-appearance: none;
+    appearance: none;
+    --slider-fill: 0%;
+    width: 100%;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)